var/spack/repos/builtin/packages/daos/package.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Daos(SConsPackage):
+    """The Distributed Asynchronous Object Storage (DAOS) is an open-source
+    software-defined object store designed from the ground up for massively
+    distributed Non Volatile Memory (NVM)."""
+
+    homepage = "https://github.com/daos-stack/daos"
+    git = "https://github.com/daos-stack/daos.git"
+    maintainers("hyoklee")
+
+    version("master", branch="master", submodules=True)
+    version("2.2.0", tag="v2.2.0", submodules=True)
+    version("2.0.2", tag="v2.0.2", submodules=True)
+    version("1.2.0", tag="v1.2.0", submodules=True)
```

Comment:
same comment here, there's no reason I think for having all these versions, I'd just start from 2.0.2 and remove all the rest. That will allow you also to remove all the other patches :) the dependency on cart  and a whole bunch of other stuff.